test/test_torch.py

```diff
@@ -6456,6 +6456,70 @@ def test_isfinite_isinf_isnan_int(self, device, dtype):
         self.compare_with_numpy(torch.isinf, np.isinf, vals, device, dtype)
         self.compare_with_numpy(torch.isnan, np.isnan, vals, device, dtype)
 
+    @unittest.skipIf(not TEST_NUMPY, 'NumPy not found')
+    @dtypes(*(torch.testing.get_all_fp_dtypes(include_bfloat16=False)))
+    def test_isposinf_float(self, device, dtype):
+        vals = (-float('inf'), float('inf'), float('nan'), -1, 0, 1)
+        self.compare_with_numpy(torch.isposinf, np.isposinf, vals, device, dtype)
+
+        t = torch.tensor(vals, device=device, dtype=dtype)
+        t_target = torch.tensor([0, 1, 0, 0, 0, 0], device=device, dtype=dtype)
+
+        out = torch.empty_like(t)
+        torch.isposinf(t, out=out)
+
+        self.assertEqual(out, t_target, msg='isposinf_out device={} dtype={}'.format(device, dtype))
```

Comment:
I wouldn't bother with a message. Adding a message will clobber the default error messages, which can be very informative.

Reply:
Deleted.